hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -2160,6 +2167,13 @@ public boolean isMetastoreEnabled() {
     return metastoreConfig.enableMetastore();
   }
 
+  /**
+   * Table Manager configs.
+   */
+  public boolean isTableManagerEnabled() {
+    return tableManagerConfig.enableTableManager();
```

Comment:
again let's make sure the name is finalized: Table Management Service, or Table Manager Service, or Table Manager?